src/vimeo/players/VimeoDesktopArticleVideoPlayer.tsx

```diff
@@ -2,84 +2,65 @@ import React, { useRef, useState } from 'react';
 import { VimeoArticleVideoPlayerProps } from 'vimeo/types';
 import useOnScreen from 'utils/useOnScreen';
 import PlayerWrapper from 'vimeo/players/shared/PlayerWrapper';
-import WDSVariables from '@fandom-frontend/design-system/dist/variables.json';
 import CloseButton from 'vimeo/players/shared/CloseButton';
 import VimeoPlayerWrapper from 'vimeo/players/shared/VimeoPlayerWrapper';
-import styled, { css, keyframes } from 'styled-components';
 
-const VimeoDesktopArticleVideoTopPlaceholder = styled.div`
-	position: absolute;
-	width: 100%;
-	padding-top: 56.25%;
-	top: 0;
-	left: 0;
-	bottom: 0;
-	right: 0;
-	z-index: ${Number(WDSVariables.z2)};
-`;
+import clsx from 'clsx';
 
-const moveDownAnimation = (right: number, width: number) => keyframes`
-	from {
-		right: ${right}px;
-		bottom: 100%;
-		width: ${width}px;
-	}
-
-	to {
-		right: 18px;
-		bottom: 45px;
-		width: 300px;
-	}
-`;
+import styles from './VimeoDesktopArticleVideoPlayer.module.css';
 
 interface VimeoDesktopArticleVideoWrapperProps {
-	isScrollPlayer: boolean;
-	right?: number;
-	width?: number;
+	vimeoDetails: any;
 }
 
-const DesktopArticleVideoWrapper = styled.div<VimeoDesktopArticleVideoWrapperProps>`
-	height: max-content;
-	${(props) =>
-		props.isScrollPlayer
-			? css`
-					z-index: ${Number(WDSVariables.z2) + 2};
-					position: fixed;
-					animation: ${moveDownAnimation(props.right, props.width)} 0.4s normal forwards;
-			  `
-			: css`
-					position: absolute;
-					bottom: 0;
-					right: 0;
-					top: 0;
-					left: 0;
-					transform: translateZ(0);
-					-webkit-transform: translateZ(0);
-			  `}
-`;
-
-const TopBar = styled.div`
-	width: 100%;
-	position: relative;
-`;
+const TopBar: React.FC = () => <div className={styles.topBar} />;
 
-const VimeoDesktopArticleVideoPlayer: React.FC<VimeoArticleVideoPlayerProps> = ({ vimeoDetails }) => {
+const DesktopArticleVideoWrapper: React.FC<VimeoDesktopArticleVideoWrapperProps> = ({ vimeoDetails }) => {
```

Comment:
Don't we want keep the variable name in the original version: `VimeoDesktopArticleVideoPlayer`? :)

Reply:
Maybe not because it's consistent with the youtube takeover 🤔 https://github.com/Wikia/jwplayer-fandom/pull/166/files#diff-42915c2ff0fcf8a86ab89d43bb0921e1bdbdbb2efce77ef16740e733eed20933R16

Reply:
Yeah, plus there is already a component named VimeoDesktopArticleVideoPlayer